apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx

```diff
@@ -46,7 +46,7 @@ const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onN
 					<Title>{ __( 'Select a plan', 'full-site-editing' ) }</Title>
 					<SubTitle>
 						{ __(
-							'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 30 days.',
+							'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 14 days.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.